src/api/authorize.ts

```diff
@@ -57,18 +57,18 @@ export default function authorize(
     );
   }
 
-  const codeChallenge =
-    codeVerifier &&
-    encode(createHash('sha256').update(codeVerifier).digest('base64').split('=')[0]);
+  const codeChallenge = codeVerifier
+    ? encode(createHash('sha256').update(codeVerifier).digest('base64').split('=')[0])
+    : undefined;
 
   const queryString = stringify({
     client_id: clientId,
     cobrand_client_id: cobrandClientId,
-    response_type: 'code',
+    response_type: codeVerifier ? 'code' : 'token',
```

Comment:
we are still not supporting implicit flow, this PR should be about not requiring PKCE to use code grant.